flow/project.py

```diff
@@ -128,7 +128,7 @@ def false(cls, key):
 
     @classmethod
     @deprecated(
-        deprecated_in="0.9", removed_in="1.0",
+        deprecated_in="0.9", removed_in="0.11",
```

Comment:
Just curious, is the reason for removing in 0.11 here vs 0.10 as most others being that we usually make deprecated and removed version to be 2 versions apart?

Reply:
Yes, that is why. We have a deprecation cycle of 2 minor versions: https://docs.signac.io/projects/flow/en/latest/support.html#deprecation-policy